public/pages/DetectorResults/components/DetectorState/DetectorInitializationFailure.tsx

```diff
@@ -35,7 +35,7 @@ export const DetectorInitializationFailure = (
           <EuiIcon type="alert" size="l" color="danger" />
 
           <h2>
-            {`The detector cannot be initialized due to ${props.failureDetail.cause}`}
+            {`The detector is not initialized because ${props.failureDetail.cause}`}
```

Comment:
@ylwu-amzn I think all of the causes start with `of` to finish this sentence here